<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgColorSpaceSelectorAdvanced</class>
 <widget class="QWidget" name="WdgColorSpaceSelectorAdvanced">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>403</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="5" column="2">
    <widget class="QDialogButtonBox" name="bnOK">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QPlainTextEdit" name="textProfileDescription"/>
   </item>
   <item row="0" column="1" rowspan="2" colspan="2">
    <widget class="QLabel" name="lbl_descriptionwidget">
     <property name="text">
      <string>Color Spaces determine how colors are encoded and thus what the result of Krita's color maths(like blending and filters) should look like. All spaces have advantages and disadvantages, so this widget will help you choose the right space for the right job.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" rowspan="2">
    <widget class="KisCIETongueWidget" name="TongueWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>400</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2" rowspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Profile Properties</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="lbltrc">
        <property name="text">
         <string>TRC</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblWhitepoint">
        <property name="text">
         <string>White Point</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblRed">
        <property name="text">
         <string>Red</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lblXYZ_W">
        <property name="text">
         <string>xyz</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lblXYZ_R">
        <property name="text">
         <string>xyz</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblGreen">
        <property name="text">
         <string>Green</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblBlue">
        <property name="text">
         <string>Blue</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="lblXYZ_G">
        <property name="text">
         <string>xyz</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="lblXYZ_B">
        <property name="text">
         <string>xyz</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" rowspan="6">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Profile Picker</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Model</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Depth</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QListWidget" name="lstProfile">
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KisCmbIDList" name="cmbColorModels"/>
      </item>
      <item row="1" column="1">
       <widget class="KisCmbIDList" name="cmbColorDepth"/>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="bnInstallProfile">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
  <customwidgets>
  <customwidget>
   <class>KisCmbIDList</class>
   <extends>QComboBox</extends>
   <header>widgets/kis_cmb_idlist.h</header>
  </customwidget>
  <customwidget>
   <class>KisCIETongueWidget</class>
   <extends>QWidget</extends>
   <header>widgets/kis_cie_tongue_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
